ci/infra/aws/README.md

```diff
@@ -11,6 +11,12 @@ this is done by `cloud-init` through a terraform variable called `authorized_key
 All the instances have a `root` and a `ec2-user` user. The `ec2-user` user can
 perform `sudo` without specifying a password.
 
+Only the master nodes have a public IP associated with. All the worker nodes
+are located on a private subnet.
```

Comment:
Would it make sense to have the master nodes exclusively on the private subnet as well? So the only public IP address is the LB.

Reply:
Yes, that would make sense. But it would require the user to create a bastion host, or to have our own terraform stuff handle that...

I would wait a first release to get this change done.